feature-libs/customer-ticketing/components/customer-ticketing-details/customer-ticketing-details.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, OnInit } from '@angular/core';
+import { TranslationService } from '@spartacus/core';
+import {
+  COLOR_CLOSE,
+  COLOR_OPEN,
+  STATUS_CLOSE,
+  STATUS_OPEN,
+} from '@spartacus/customer-ticketing/root';
+import { Card } from '@spartacus/storefront';
+import { Observable, of } from 'rxjs';
+import { filter, map } from 'rxjs/operators';
+
+@Component({
+  selector: 'cx-customer-ticketing-details',
+  templateUrl: './customer-ticketing-details.component.html',
+})
+export class CustomerTicketingDetailsComponent implements OnInit {
+  constructor(protected translation: TranslationService) {}
+
+  ticketDetails$ = of({
+    associatedTo: {
+      code: '00000001',
+      modifiedAt: '2022-06-28T00:00:00+0000',
+      type: 'Cart',
+    },
+    availableStatusTransitions: [
+      {
+        id: 'CLOSED',
+        name: 'Closed',
+      },
+    ],
+    createdAt: '2022-06-22T14:37:15+0000',
+    id: '00000001',
+    modifiedAt: '2022-06-22T20:25:02+0000',
+    status: {
+      id: 'OPEN',
+      name: 'Open',
+    },
+    subject: 'test ticket again',
+    ticketCategory: {
+      id: 'COMPLAINT',
+      name: 'Complaint',
+    },
+    ticketEvents: [
+      {
+        author: 'Mark Rivers',
+        createdAt: '2022-06-22T20:25:02+0000',
+        message: 'This is the way',
+      },
+      {
+        author: 'Mark Rivers',
+        createdAt: '2022-06-22T14:37:15+0000',
+        message: 'A message to consider',
+      },
+    ],
+  });
+
+  ngOnInit(): void {}
+
+  prepareCardContent(
+    entity: string,
+    titleTranslation: string
+  ): Observable<Card> {
+    return this.translation.translate(titleTranslation).pipe(
+      filter(() => Boolean(entity)),
+      map((textTitle) => ({
+        title: textTitle,
+        text: [entity],
+        customCss: this.getStatusColor(entity),
+      }))
+    );
+  }
+
+  getStatusColor(status: string): { color: string } | null {
```

Comment:
related to the constants file

I see you went with the styling approach, but what do you think about classes instead. I believe it's more configurable in a sense that a customer can override the class in the css level

If going with classes, maybe no need for the enums anymore

Reply:
> By the way, not really relevant to this PR, but can you merge pull changes to develop and merge it to your epic.
> 
> Reason being there is a fix for 2205 / 2211 servers, where it broke PLP page in Spartacus

Yup. Noted.